libs/platform/src/lib/shared/domain/data-source.ts

```diff
@@ -181,7 +185,7 @@ export class ComboBoxDataSource<T> implements DataSource<T> {
             throw new Error('DataSource.match() predicate can only accepts string and Map');
         }
 
-        if (!searchParam.has('limit')) {
+        if (!searchParam.has('limit') && !this.limitless) {
```

Comment:
I mean, we can keep `@Input() limitless` in component and if it's not provided then use the default limit. But here, in data-source i think just one of them is enough.

Reply:
@platon-rov do you offer to throw away `ComboBoxDataSource.MaxLimit` ? because, if I remove `&& !this.limitless` and consider `!searchParam.has('limit')` means limitless, than this max limit is an obstacle here

Reply:
We have limitless input property in component, we can set it to false by default so we will limit results to certain number using `searchParam.set('limit', ...)`. 

But user will be able to set limitless to true then we won't be setting this map key-value pair. That's  what i meant.